config/default.yml

```diff
@@ -523,6 +523,15 @@ Style/MultilineOperationIndentation:
 Style/NumericLiterals:
   MinDigits: 5
 
+Style/OptionHash:
+  # A list of parameter names that will be flagged by this cop.
+  InvalidParameters:
```

Comment:
I wouldn't say those params are invalid - there are just likely names for option hash params. Maybe this should be named "RelevantParamNames" or something like this. @jonas054 better ideas welcome.